### ISSUE TITLE
Number of training data

### ISSUE THREAD

**Issue by wwgjob** [ZZY-Zhou/RENet#6]
epoch is  1
default/home/chris/anaconda3/envs/renet/lib/python3.11/site-packages/torch/nn/_reduction.py:42: UserWarning: size_average and reduce args will be deprecated, please use reduction='sum' instead.
  warnings.warn(warning.format(ret))
default |################################| train: [1][8479/8480]|Tot: 1:21:32 |ETA: 0:00:01 |loss 2.7975 |loss_hm 2.0930 |loss_mov 0.2979 |loss_wh 4.0665 

I saw the number of the training dataset was 8480 but the README wrote is 10495.
And I didn't change the DSEC-GT.pkl.
It is normal?

**Comment by Zongwei97** [comment]
Hi,
Thanks for your interest in our paper.
Our dataset is composed of 10495 frames. You can verify the number in the dataset directly.
I think the 8480 is related to your batch size used for training.
Best
